jit-compiler/src/codegen.rs

```diff
@@ -354,25 +354,43 @@ fn map_type(ty: &Type) -> String {
     }
 }
 
-fn is_builtin(symbol: &str) -> bool {
-    matches!(
-        symbol,
-        "std::check::panic" | "std::field::modulus" | "std::convert::fe"
-    )
+fn get_builtins<T: FieldElement>() -> &'static HashMap<String, String> {
+    static BUILTINS: OnceLock<HashMap<String, String>> = OnceLock::new();
+    BUILTINS.get_or_init(|| {
+        [
+            (
+                "std::array::len",
```

Comment:
is `std::convert::fe` missing on purpose?

Reply:
It was not working as it was written any way. Will be re-added in https://github.com/powdr-labs/powdr/pull/1834